rasa/core/domain.py

```diff
@@ -71,6 +74,15 @@ def get_exception_message(
             if message:
                 message += "\n"
             message += get_mapping_exception_message(mappings)
+        if templates:
+            for template in templates:
+                if message:
+                    message += "\n"
+                message += (
+                    "Utterance template '{}' is listed in as an "
+                    "action in the domain file, but there is "
+                    "no matching utterance template"
```

Comment:
```suggestion
                    "no matching utterance template."
```
should be printed in warning yellow